src/convertToObject.js

```diff
@@ -12,7 +12,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stringArr = sourceString.split(';');
+
+  const formatArr = stringArr
+    .map(element => element
+      .split(':')
+      .map(stringItem => stringItem.trim()))
+    .filter(item => item.length > 1);
```

Comment:
```suggestion
  const formatArr = sourceString.split(';')
    .map(element => element
      .split(':')
      .map(stringItem => stringItem.trim()))
    .filter(item => item.length > 1);
```

No need to create one more variable. You could chain all methods